package.json

```diff
@@ -49,7 +48,7 @@
   },
   "private": true,
   "dependencies": {
```

Comment:
"development" related dependencies should be defined under "devDependencies". Right now, everything is clubbed into a single section.

Reply:
This was the project structure since we migrated to the `nx`. I'm not sure about which dependencies should I move in the `devDependencies` section. Can you please guide me on that?

Reply:
Any dependency that is consumed by the SDK and finally part of the distributable should be listed under `dependencies`. Everything else should be in `devDependencies` as they support the project development, testing, packaging, and distribution.

Reply:
I’ll create a separate ticket for this change to move the development dependencies to the correct place.
CC: @ChryssaAliferi

Reply:
Here is a card for that: https://linear.app/rudderstack/issue/SDK-2138/move-the-development-dependencies-to-the-devdependencies-section-in-rn.